devbox.go

```diff
@@ -137,13 +137,8 @@ func (d *Devbox) Build(flags *docker.BuildFlags) error {
 
 // Plan creates a plan of the actions that devbox will take to generate its
 // shell environment.
-func (d *Devbox) ShellPlan() (*plansdk.Plan, error) {
-	userPlan := d.convertToPlan()
-	shellPlan, err := planner.GetShellPlan(d.srcDir)
-	if err != nil {
-		return nil, err
-	}
-	return plansdk.MergeUserPlan(userPlan, shellPlan)
+func (d *Devbox) ShellPlan() *plansdk.Plan {
+	return d.convertToPlan()
```

Comment:
Could you add a TODO to make this a different struct at a later point? (it makes no sense for the "shell plan" to still include things like build stage)